VueJSProject/gab560/add.php

```diff
@@ -102,7 +102,7 @@
 	</textarea>
 	<br>
    
-   <input class="btn btn-primary" :disabled=isDisabled type="button" id="submit" v-on:click:"add">Add Device</button>
+   <button class="btn btn-primary" :disabled=isDisabled type="button" id="submit" v-on:click:"add">Add Device</button>
```

Comment:
<button class="btn btn-primary" :disabled=isDisabled id="submit" type="submit">Add Device</button>

fixes it. Thanks